### DISCUSSION TITLE
Icon request: label

### DISCUSSION THREAD

**Discussion by yap-studios** [FortAwesome/Font-Awesome#21125] [Icon Requests]
**What can it be used for?**
It is used to represent a material design label.

**Example image (optional)**
![download](https://user-images.githubusercontent.com/46919661/69721641-f5930c00-111d-11ea-9cd0-3a15b12a06e3.jpeg)

**Icon request checklist**
- [ x ] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [ x ] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [ x ] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [ x ] I have included at least one example image if it could be helpful (optional)
- [ x ] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Released in 7.2: https://fontawesome.com/icons/label?f=classic&s=solid
